model/monster.html

```diff
@@ -17,7 +17,8 @@
                                 data-dtype="Number" />
                         </div>
                         <div class="armor">
-                            <label><button type="button" class="roll-armor">{{localize data.armor.label}}</button>:</label>
+                            <label><button type="button"
```

Comment:
Just out of curiosity: autoformat struck again? 😄

Reply:
Yeah I have format on save. But this seems unintended. No reason to push changes in this file. I'll do a hard reset before merging, will make sure this file isn't committed again.